### ISSUE TITLE
Using Wit in eclipse without gradle

### ISSUE THREAD

**Issue by harveyslash** [wit-ai/wit-android-sdk#14]
Hello, I want to use Wit for android in Eclipse. However, I dont know gradle. Could you please help me out with that? I have also asked a question in stackoverflow.
http://stackoverflow.com/questions/29408228/wit-ai-in-android-and-eclipse-nothing-happening

**Comment by jianwue** [comment]
Hello, do you have find out your problem? I build it with eclipse also, and it run in my android device normally, but when i send a message to the server for NLU, an exception occurred, the exception is "An error occurred during the request, did you set your token correctly?", could you help me? Thank you.

**Comment by harveyslash** [comment]
wit requires that you put your api key in the library. Have you done that?

**Comment by jianwue** [comment]
Yes, I set my Server Access Token in function “onCreate()”,below is my codes:

  @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        String accessToken = "EYNC3HOMVYETCTN7LV2D74TTG2OZXNV5";//serverAccessToken
        _wit = new Wit(accessToken, this);
        _wit.enableContextLocation(getApplicationContext());

        String text = "Wake me up at 6am";
        _wit.captureTextIntent(text);
}

Thanks.

**Comment by harveyslash** [comment]
I think you have given the server access token. You have to provide you
client access token. Can be found in the settings

**Comment by l5t** [comment]
The Client Access Token is only needed for the web sdk. So you should put your Server Access Token here
